Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Task.java

```diff
@@ -91,6 +91,9 @@
     @Column(name = "typeAcceptClose")
     private boolean typeAcceptClose = false;
 
+    @Column(name = "typeExportable")
+    private boolean typeExportable = false;
```

Comment:
why typeExportable and not "exportable"?

Reply:
All these boolean properties within Task are called starting with type. I just wanted to follow the convention. Should it only be called exportable? I think the name is confusing, because the task itself is not exportable. Maybe we need a better name. Maybe `typeExportableWhenDone`? I was here but first to the concept of how to save it. Do you think that's right?

Reply:
After consultation with Sebastian that should be solved differently. The “exportable” property is said to be a process property, and the step should only set this property to `true` when it is reached. (It should also be possible to manually change this process property.) So I would suggest here as a name: `typeSetExportableToTrue`.